### PR TITLE
installation: separate COD and Invenio development packages

### PR DESCRIPTION
**Pull request by tiborsimko**
* Separates CERN Open Data and Invenio required development packages so that one
  can use e.g. bleeding edge CERN Open Data packages with PyPI-released Invenio
  packages for the overlay development.

Signed-off-by: Tibor Simko <tibor.simko@cern.ch>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -50,7 +50,7 @@ before_install:
   - "travis_retry pip install twine wheel coveralls requirements-builder"
   - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
   - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=dev --req requirements-devel-cernopendata.txt setup.py > .travis-devel-requirements.txt"
```
Comment (jirikuncar) [inline .travis.yml]: We should then add separate matrix entry for Invenio devel packages to detect early failures.

Comment (tiborsimko) [reply .travis.yml]: Yes, and I was also going to separate `docker-compose.yml` from `docker-compose-dev.yml`later...

Comment (tiborsimko) [reply .travis.yml]: ... and note that this separation requires to make a new release of `dojson`. I'll prepare a PR.

Comment (tiborsimko) [reply .travis.yml]: All done.

**Review by jirikuncar — APPROVED** [review]

**Pull request merged**